octoprint_mrbeam/static/js/render_fills.js

```diff
@@ -64,6 +66,7 @@ Snap.plugin(function (Snap, Element, Paper, global) {
                 elem.type === "text" ||
                 elem.type === "#text"
             ) {
```

Comment:
This if condition is useless right now and only adds cognitive complexity... We could remove it.
It might also be good to move all the else branch corresponding to the `if (children.length > 0 && goRecursive) {` to a separate method which could be called processElementByType(type)

Reply:
Done